src/modules/sched-simple/test/rlist.c

```diff
@@ -202,6 +202,74 @@ static void test_simple (void)
     rlist_destroy (copy);
 }
 
+const char R_issue2202[] = "{\
+  \"0\": {\
+    \"Package\": 1,\
+    \"Core\": 1,\
+    \"PU\": 1,\
+    \"cpuset\": \"0\"\
+  },\
+  \"1\": {\
+    \"Package\": 1,\
+    \"Core\": 1,\
+    \"PU\": 1,\
+    \"cpuset\": \"1\"\
+  },\
+  \"2\": {\
+    \"Package\": 1,\
+    \"Core\": 1,\
+    \"PU\": 1,\
+    \"cpuset\": \"2\"\
+  },\
+  \"3\": {\
+    \"Package\": 1,\
+    \"Core\": 1,\
+    \"PU\": 1,\
+    \"cpuset\": \"3\"\
+  }\
+}";
+
+static void test_issue2202 (void)
+{
+    char *result = NULL;
+    struct rlist *a = NULL;
+
+	struct rlist *rl = rlist_from_hwloc_by_rank (R_issue2202);
```

Comment:
nit: used tabs instead of spaces

Reply:
ugh, sorry about that. Somehow the editor I was using seems to have ignored modeline...